tests/testthat/test-layers.R

```diff
@@ -458,7 +458,18 @@ test_call_succeeds("bidirectional", {
     layer_activation(activation = "softmax")
 })
 
+test_call_succeeds("layer_activation_softmax", {
```

Comment:
If these are functions only available in the latest version of Keras, you also need to add the `required_version` parameter to `test_call_succeeds()` to ensure that they are only run if Keras >= 2.1.3 is available (e.g. https://github.com/rstudio/keras/blob/master/tests/testthat/test-layers.R#L143-L148)